htdocs/core/lib/company.lib.php

```diff
@@ -1330,6 +1330,7 @@ function show_actions_done($conf, $langs, $db, $filterobj, $objcon='', $noprint=
         $sql.= " FROM ".MAIN_DB_PREFIX."actioncomm as a";
         $sql.= " LEFT JOIN ".MAIN_DB_PREFIX."user as u on u.rowid = a.fk_user_action";
         $sql.= " LEFT JOIN ".MAIN_DB_PREFIX."c_actioncomm as c ON a.fk_action = c.id";
+        $sql.= " LEFT JOIN ".MAIN_DB_PREFIX."actioncomm_resources as r ON a.id = r.fk_actioncomm";
```

Comment:
If you add this here and objcon is not set as entry parameter, the request will return several line when we expect to have one line per action.

Reply:
Hello @eldy 

Thank you for spotting the error when no `$objcon` is given. The condition has been added.


> Also, we should add a DISTINCT when $objcon is provided to avoid to have loop later doubled.

The tests report no Agenda events were doubled in the SQL results.

Still, a `SELECT DISTINCT a.id, a.label,` was added as you suggest.